### DISCUSSION TITLE
How does Snipe-IT get assets in?

### DISCUSSION THREAD

**Discussion by ghost** [snipe/snipe-it#11340] [General]
Does Snipe-IT have a client program on the external clients which then talk to the server to populate the inventory?

**Comment by mikeroq** [comment]
There are a few ways AFAIK (correct me if I'm wrong):

- A third party Windows agent - https://github.com/Scope-IT/marksman
- Manually creating assets
- Importing from CSV
- Using the API (what the third party agent does)

**Comment by phil-flip** [comment]
There is also an Android (free) and an IOS (~$4) app.
They seem to work fine, but I haven't used snipe-it for production yet.

Alternatively, you can always fall back to the native web interface
